packages/catalog-process/test/db.test.ts

```diff
@@ -2202,7 +2204,7 @@ describe("database test", async () => {
     });
 
     describe("getEServiceById", () => {
-      it("should get the eService if it exists", async () => {
+      it("should get the eService if it exists (requester is the producer, admin)", async () => {
```

Comment:
Just to be careful, could you add a happy path test for `api,security`?

Reply:
I can add it, if you prefer.
Anyways, this logic is pretty straightforward: the condition `authData.userRoles.includes(userRoles.API_ROLE)` will be satisfied if the array of roles contains at least `userRoles.API_ROLE` (so in case the `authToken` has both `userRoles.API_ROLE` and `userRoles.SECURITY_ROLE` it will pass the condition). The real test would be to see if the auth middleware turns `"api,security"` into an array of `[userRoles.API_ROLE, userRoles.SECURITY_ROLE]`, but there is no way to assert that from here. If I set the `userRoles` array from here, it will of course pass the test, but we will not know whether the array will be passed like so by the `authMiddleware`, which is not involved in this test.